File: app/src/main/java/kr/gyeongmunbaek/imagesequenceanimaion/view/image/SampleBitmapCreator.java
package kr.gyeongmunbaek.imagesequenceanimaion.view.image;

import android.content.Context;

/**
 * Created by gyeongmunbaek on 2017-10-26.
 */

public class SampleBitmapCreator extends BitmapCreatorParent {
    public SampleBitmapCreator() {
        super();
    }

    public SampleBitmapCreator(Context pContext, Integer pIndex) {
        super(pContext, pIndex);
    }

    @Override
    public int getFileCount() {
        return 22;
    }

    @Override
    public String getFileNamePrefix() {
        return "output_";
    }

    @Override
    public int getStartIndex() {
        return 0;
    }
}

File: app/src/main/java/kr/gyeongmunbaek/imagesequenceanimaion/view/image/SampleBitmapCreator.kt
package kr.gyeongmunbaek.imagesequenceanimaion.view.image

import android.content.Context

/**
 * Created by gyeongmunbaek on 2017-10-26.
 */

class SampleBitmapCreator : BitmapCreatorParent {

    override val fileCount: Int = 22
    // set prefix string of image file name
    override val fileNamePrefix: String = "output_"
    // set images start file index. ex) in output_0"0"
    override val startIndex: Int = 0

    constructor() : super()

    constructor(context: Context, index: Int) : super(context, index)
}
